macro/makeGenieEvents.py

```diff
@@ -11,7 +11,8 @@
 # Before runnig this script please run this command in FairShip bash:
 # export GXMLPATH='/eos/experiment/ship/user/aiuliano/GENIE_FNAL_nu_splines'
 # this will disable Genie decays for charm particles and tau
-
+if os.environ.has_key('GXMLPATH'): print('exporting GXMLPATH executed')
```

Comment:
We should probably not print anything if this is correct (or print a debug message, which would be silent by default unless the verbosity is increased).